### DISCUSSION TITLE
Asymmetric routing / userdefined route on deployment

### DISCUSSION THREAD

**Discussion by RobertAmdax** [Azure/AKS-Construction#300] [Q&A]
I am happily using the AKS construction helper - awesome work! 
One issue we are running into is restricting egress through a central Azure Firewall, which causes asymmetric routing. Like described [here](https://docs.microsoft.com/en-us/azure/aks/limit-egress-traffic)
The solution would be deploying AKS with the **--outbound-type userDefinedRouting**  option on deployment. 
Does anyone have a hint on how to do this with the AKS construction helper? 
Thanks,
Robert

**Comment by Gordonby** [comment]
Closed by #550
